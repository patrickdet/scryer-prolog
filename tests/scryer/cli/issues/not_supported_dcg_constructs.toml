# issue 2675
args = [
    "-f",
    "--no-add-history",
    "-g", "halt",
     "main.pl"
]
