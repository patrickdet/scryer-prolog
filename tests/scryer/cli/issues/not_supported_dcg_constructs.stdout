   error(representation_error(dcg_body),[culprit-({true}->[])]).
